devicetypes/smartthings/rgbw-light.src/rgbw-light.groovy

```diff
@@ -352,6 +357,11 @@ def isAeotecLed6() {
 	 || (zwaveInfo?.mfr?.equals(AEOTEC_LED6_MFR) && zwaveInfo?.prod?.equals(AEOTEC_LED6_PROD_EU) && zwaveInfo?.model?.equals(AEOTEC_LED6_MODEL)))
 }
 
+def isAeotecLedStrip(){
```

Comment:
I think just checking for manufacturer and model is enough here.

Reply:
Ok